internal/db/db_sqlite.go

```diff
@@ -61,8 +56,8 @@ type SqliteBackend struct {
 	DB *sql.DB
 }
 
-func New() (DatabaseBackend, error) {
-	db, err := sql.Open(driverName, file)
+func New(driverName string, url string) (DatabaseBackend, error) {
```

Comment:
```suggestion
func New(driverName, url string) (DatabaseBackend, error) {
```
Nitpick, but more go like? You can actually group params of the same type together like this.